### ISSUE TITLE
Incorrect example ...

### ISSUE THREAD

**Issue by spock64** [publichealthengland/coronavirus-dashboard-api-javascript-sdk#3]
The example structure parameters are invalid, resulting in a 404.

You should be using newDeaths28DaysByDeathDate and cumDeaths28DaysByDeathDate

**Comment by xenatisch** [comment]
Yes, I am aware. We have been busy incorporating them into the pipeline, the ETL, the API, the developer's guide, and create the underlying infrastructure need to publish them... not to mention the new deaths page. So far, I have updated the metrics for the R package. I'll get to the rest of them.

We are working round the clock to provide and maintain the service. I'm afraid there's just not enough hours in a day! We'll get there.
